packages/editor/src/extensions/table/table.ts

```diff
@@ -477,19 +477,19 @@ export const Table = Node.create<TableOptions>({
   },
 
   addProseMirrorPlugins() {
-    const isResizable = this.options.resizable && this.editor.isEditable;
+    // const isResizable = this.options.resizable && this.editor.isEditable;
 
     return [
-      ...(isResizable
-        ? [
-            columnResizing({
-              cellMinWidth: this.options.cellMinWidth,
-              View: TableNodeView(this.editor),
-              showResizeHandleOnSelection:
-                this.options.showResizeHandleOnSelection
-            })
-          ]
-        : [tiptapTableView(this.options.cellMinWidth)]),
+      // ...(isResizable
+      // [
+      columnResizing({
+        cellMinWidth: this.options.cellMinWidth,
+        View: TableNodeView(this.editor),
+        showResizeHandleOnSelection: this.options.showResizeHandleOnSelection,
+        isResizable: this.options.resizable
+      }),
+      // ],
+      // : [tiptapTableView(this.options.cellMinWidth)]),
```

Comment:
I think we should allow column resizing in a readonly editor. It is similar action to collapse/expand headings and the state isn't preserved in the note's content.

Though I'm not sure about the significance of rendering the `TableView` in `tiptapTableView` when we can just render the `TableNodeView` and make some tweaks to handle to handle readonly case.

@thecodrr what do you think about this?